spec/system/consumer/split_checkout_spec.rb

```diff
@@ -627,6 +627,7 @@
             it "selects Stripe SCA and proceeds to the summary step" do
               choose pay_method.to_s
               fill_out_card_details
+              click_on "Next - Order summary"
```

Comment:
`yes, this is a hack, but can't make it work as a normal behavior. For testing purpose, we use `spec/support/fixtures/stripejs-mock.js` and it seems not working with our `app/webpacker/controllers/stripe_controller.js``

Yeah! I've had troubles with this too... I'm not fully understanding this yet, and the implications but at some point good it would be great to exchange thoughts on this.